docs/README.md

```diff
@@ -0,0 +1,8 @@
+## Documentation for SSI DIM Wallet stub application
+
+#### Index
+
+1. How to contribute: [How to contribute.md](admin%2FDev%20Process%2FHow%20to%20contribute.md)
+2. Open API documentation: [openAPI.json](api%2FopenAPI.json)
+3. Architecture Documentation: [arc42](architecture%2Fmain.md)
+4. Integration with other components: [integration.md](admin%2Fintegration%2Fintegration.md)
```

Comment:
I think this index is a bit weird as it does reflect the contents of /docs partially/very selectively

Reply:
Modified with: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/2/commits/7ce8855d202a91dd166b799f989ac15740b98843

Let me know if further changes are necessary